config/findbugs-exclude.xml

```diff
@@ -53,6 +53,17 @@
         <Bug pattern="SE_COMPARATOR_SHOULD_BE_SERIALIZABLE"/>
     </Match>
 
+    <!-- Special handling of clone method for cloneable BsonValue subclasses.  By design, they don't call super.clone() -->
+    <Match>
+        <Or>
```

Comment:
I don't trust findbugs-exclude - previously using a nested match meant nothing was checked for that code.

In the end we had to explode it out into multiple statements.